packages/components/psammead-caption/src/index.jsx

```diff
@@ -19,7 +19,7 @@ const Caption = styled.figcaption`
   color: ${C_CLOUD_DARK};
   font-family: ${GEL_FF_REITH_SANS};
   ${FS_ITALIC};
-  padding: ${GEL_SPACING} 0;
+  padding: ${GEL_SPACING};
   width: 100%;
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
     padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
```

Comment:
Could you please fix forward instead of reverting?

```suggestion
    padding: ${GEL_SPACING} 0;
```

Reply:
Happy to do so provided we know what we're doing! I tried this locally and it looked odd to me. Given we're launching Mon/Tues, wanted to get it back from "badly broken on mobile" to "looks a bit weird on desktop".

Does this line definitely fix it? 😀